src/traces/bar/style_defaults.js

```diff
@@ -23,6 +23,10 @@ module.exports = function handleStyleDefaults(traceIn, traceOut, coerce, default
 
     coerce('marker.line.width');
     coerce('marker.opacity');
+    coerce('marker.pattern.shape');
+    coerce('marker.pattern.bgcolor');
+    coerce('marker.pattern.size');
+    coerce('marker.pattern.solidity');
```

Comment:
```suggestion
    var pattern = coerce('marker.pattern.shape');
    if (pattern) {
        coerce('marker.pattern.bgcolor');
        coerce('marker.pattern.size');
        coerce('marker.pattern.solidity');
    }
```
We only want to coerce attribute we will use :)